src/iSparrow/utils.py

```diff
@@ -131,17 +157,63 @@ def load_model_from_file_torch(path: str, _):
     raise NotImplementedError("torch models are not yet supported")
 
 
-def load_module(alias: str, path: str):
+def load_module(module_name: str, file_path: str):
     """
     load_module Load a python module from 'path' with alias 'alias'
 
     Args:
-        path (str): Path to load the module from
+        module_name (str): module alias.
+        file_path (str): Path to load the module from
 
     Returns:
         module: Python module that has been loaded
     """
-    spec = importlib.util.spec_from_file_location(alias, path)
-    module = importlib.util.module_from_spec(spec)
-    spec.loader.exec_module(module)
+    try:
```

Comment:
this needed a better exception handling... was missing before and was overlooked.